org.lflang.tests/META-INF/MANIFEST.MF

```diff
@@ -7,13 +7,13 @@ Bundle-Version: 0.2.2.qualifier
 Bundle-SymbolicName: org.lflang.tests; singleton:=true
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.lflang,
- org.junit.jupiter.api;bundle-version="[5.0.0,6.0.0)",
+ org.junit.jupiter.api;bundle-version="5.8.2",
  org.eclipse.xtext.testing,
  org.eclipse.xtext.xbase.testing,
- org.eclipse.xtext.xbase.lib;bundle-version="2.26.0",
+ org.eclipse.xtext.xbase.lib;bundle-version="2.27.0",
  javax.annotation;bundle-version="1.2.0",
  org.eclipse.core.jobs,
  org.opentest4j;bundle-version="1.2.0",
- org.eclipse.lsp4j;bundle-version="0.10.0"
+ org.eclipse.lsp4j;bundle-version="0.12.0"
```

Comment:
It's a little weird that the LSP4J version was lagging behind in the tests package. It makes me wonder if LSP4J is even needed in there.

Reply:
Yeah, it's a little weird.